docs/plugins/cloud-gce.asciidoc

```diff
@@ -46,6 +46,48 @@ discovery:
       type: gce
 --------------------------------------------------
 
+
+[IMPORTANT]
+.Binding the network host
+==============================================
+
+It's important to define `network.host` as by default it's bound to `localhost`.
+
+You can use {ref}/modules-network.html[core network host settings] or
+<<discovery-gce-network-host,gce specific host settings>>:
+
+==============================================
+
+[[discovery-gce-network-host]]
+==== GCE Network Host
+
+When the `cloud-gce` plugin is installed, the following are also allowed
+as valid network host settings:
+
+[cols="<,<",options="header",]
+|==================================================================
+|GCE Host Value |Description
+|`_gce:privateIp_` |The private IP address of the machine.
+|`_gce:hostname_` |The hostname of the machine.
+|`_gce_` |Less verbose option for the private ip address.
+|==================================================================
+
+By default, GCE discovery will use network card `0` when fetching `_gce:privateIp_`
+from metadata server. If you want to get the IP address from another network card,
+set `cloud.gce.network.card`:
+
+[source,yaml]
+--------------------------------------------------
+cloud:
+  gce:
+      project_id: <your-google-project-id>
+      zone: <your-zone>
+      network.card: 1
```

Comment:
I think it will be better to bake this into the network host. i.e., `_gce:privateIp:0_` (where 0 is the network interface)